chris/zigzag-conversion.go

```diff
@@ -0,0 +1,53 @@
+package chris
+
+import (
+	"strings"
+)
+
+func convert(s string, numRows int) string {
+
+	if len(s) == 1 || numRows == 1 {
+		return s
+	}
+
+	type grid struct {
+		row int
+		col int
+	}
+
+	zig := make([][]rune, numRows)
+	for i := range zig {
+		zig[i] = make([]rune, (len(s)/2)+1)
+	}
+
+	lastRow := 0
+	cursor := grid{0, 0}
+
+	for _, c := range s {
+		if cursor.col%(numRows-1) == 0 {
```

Comment:
Is this `%` a modulo operator here like in other langs?

Reply:
Yeah, works the same as everywhere else.